_posts/2018-07-23-pointers.md

```diff
@@ -50,7 +50,7 @@ it, you need to store it (in a variable, a field of a structure, an array cell
 uninitialised, it can become invalid ...
 
 ```c
-int x = 42;	// we need an address to init our pointer
+int x = 42;	// let's declare a variable, this will allow us later to have an address to init our pointer
```

Comment:
A bit long for a comment

Reply:
I agree, but, from the other side, I think that the actual comment can be misleading for a beginner.